testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/security/AuthenticationPolicyContextTestCase.java

```diff
@@ -0,0 +1,333 @@
+            fail("Throwable:" + t.getMessage());
+        }
+    }
+
+    @After
+    public void after() throws IOException {
+        deployer.undeploy(PICKETLINK_STS);
+        deployer.undeploy(PICKETLINK_STS_WS);
+        container.stop(CONTAINER);
+    }
+
+    @BeforeClass
+    public static void initClient() throws Exception {
+        wsClient = new WSTrustClient("PicketLinkSTS", "PicketLinkSTSPort",
+                getHttpUrl(DEFAULT_HOST, DEFAULT_PORT) + "picketlink-sts/PicketLinkSTS", new WSTrustClient.SecurityInfo(USERNAME, PASSWORD));
+    }
+
+    @AfterClass
+    public static void closeClient() throws Exception {
+        if (wsClient != null) {
+            wsClient.close();
+        }
+    }
+
+    private static String getHttpUrl(String host, int port) {
+        return "http://" + host + ":" + port + "/";
+    }
+
+    private static String getHostname() {
+        String hostname = System.getProperty("node0");
+
+        try {
+            hostname = NetworkUtils.formatPossibleIpv6Address(InetAddress.getByName(hostname).getHostAddress());
+        } catch (UnknownHostException ex) {
+            String message = "Cannot resolve host address: " + hostname;
+            throw new RuntimeException(message, ex);
+        }
+
+        if ("127.0.0.1".equals(hostname)) {
+            return "localhost";
+        }
+
+        return hostname;
+    }
+
+    private void initServerConfiguration(String deployment1, String deployment2) throws Exception {
+        modelControllerClient = ModelControllerClient.Factory.create(HOST, getManagementPort());
+        commandCtx = CLITestUtil.getCommandContext(HOST, getManagementPort(), null, consoleOut, -1);
+        commandCtx.connectController();
+
+        File cliFile = File.createTempFile("add-security-domain-", ".cli");
+        try (FileOutputStream fos = new FileOutputStream(cliFile)) {
+            IOUtils.copy(AuthenticationPolicyContextTestCase.class.getResourceAsStream("add-security-domain.cli"), fos);
+        }
+        runBatch(cliFile);
+        cliFile.delete();
+        reload();
+
+        cliFile = File.createTempFile("add-jaxws-endpoint-", ".cli");
+        try (FileOutputStream fos = new FileOutputStream(cliFile)) {
+            IOUtils.copy(AuthenticationPolicyContextTestCase.class.getResourceAsStream("add-jaxws-endpoint.cli"), fos);
+        }
+        runBatch(cliFile);
+        cliFile.delete();
+        reload();
+
+        deployer.deploy(deployment1);
+        deployer.deploy(deployment2);
+    }
+
+    /**
+     * Sends command line to CLI.
+     *
+     * @param line specifies the command line.
+     * @param ignoreError if set to false, asserts that handling the line did not result in a
+     *        {@link org.jboss.as.cli.CommandLineException}.
+     *
+     * @return true if the CLI is in a non-error state following handling the line
+     */
+    public boolean sendLine(String line, boolean ignoreError) {
+        consoleOut.reset();
```

Comment:
As per above this should just be using our standard ServerSetupTask stuff rather than CLI scripts.